microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,54 @@ To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test u
 
 
 <!-- ====================================================================== -->
+## 1.0.1369-prerelease

```

Comment:
@imicy Will there be a release H2 section in this PR?

Reply:
https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1343.22 - "a day ago"